cmd/archeio/app/buckets.go

```diff
@@ -26,7 +26,9 @@ import (
 // awsRegionToS3URL returns the base S3 bucket URL for an OCI layer blob given the AWS region
 //
 // blobs in the buckets should be stored at /containers/images/sha256:$hash
-func awsRegionToS3URL(region string) string {
+//
+// If the region is not known, returns false.
+func awsRegionToS3URL(region string) (string, bool) {
```

Comment:
this method handles ALL possible regions as enforced by a unit test, so this makes the code more complicated for no gain?